README.md

```diff
@@ -1 +1,221 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello-World] <a name="about-project"></a>
+
+
+**[Hello-World]** Is a practice where I put my knowledge about all of the tools and best practices that I learned these days. I'm testing the linters of html/css
+
+## 🛠 Built With <a name="built-with"></a>
+
+```
+1.- HTML.
+
+2.- CSS.
+
+3.- Linters.
+```
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+   </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+```
+- **[Added linters]**
+- **[Added index.html]**
+- **[Added Css.css]**
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Coming soon
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+```
+- Code Editor.
+- Git and Github account.
+- Web browser.
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+```
+  cd my-folder
+  git clone https://github.com/CesarHerr/Hello-World.git
+```
+
+
+### Install
+
+Install this project with:
+
+
+
+```
+You Don't need install this project
+
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```
+Double click on the HTML file, open it in your browser.
+
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```
+Not need it
+
+```
+
+
+### Deployment
+
+You can deploy this project using:
+
+
+[Implementation](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) 
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **César Herrera**
+
+- GitHub: [@CesarHerr](https://github.com/CesarHerr)
+- Twitter: [@Cesarherr2](https://twitter.com/Cesarherr2)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/cesarherr/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+No features
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```

Comment:
- [ Required ] Please consider removing this instructional text from your readme file, this text serves to guide you on how to fill this section of the readme file. You are supposed to remove the instructional text after filling in this section, please remove it.